src/components/CollectionsPage/CollectionRow/CollectionRow.container.ts

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux'
+import { push } from 'connected-react-router'
+import { RootState } from 'modules/common/types'
+import { getCollectionItems } from 'modules/item/selectors'
+import { OwnProps, MapStateProps, MapDispatchProps, MapDispatch } from './CollectionRow.types'
+import CollectionRow from './CollectionRow'
+
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => ({
+  items: getCollectionItems(state, ownProps.collection.id)
```

Comment:
The total amount of items is based on the items loaded in the UI, this might break with pagination, as it assumes that all the items are loaded. We could "fix" this by returning the total amount of items for the collections in the builder server.